apps/testrunner/ssh.py

```diff
@@ -2,7 +2,7 @@
 import os
 import socket
 import tempfile
-from six import StringIO
+from six import BytesIO
```

Comment:
I guess BytesIO are probably better here but could you share what is the difference in this specific situation?